### ISSUE TITLE
MediaSession: Resuming playback incorrect in console

### ISSUE THREAD

**Issue by khorn4** [muxinc/mux-stats-sdk-media3#40]
When using MediaSession and using MuxStatsSdkMedia3 to monitor the player, the events after a pause event on the video is performed are showing as "Starting Up" on the console even while video is currently playing on the device. This will happen pressing pause directly in the player or via the notification.
<img width="903" alt="Screenshot 2023-09-20 at 3 28 42 PM" src="https://github.com/muxinc/mux-stats-sdk-media3/assets/6546162/df91f316-b0ba-4629-b8d0-117e3a601be9">

This can be reproduced by cloning the androidx/media repo and using the [session demo](https://github.com/androidx/media/tree/main/demos/session). I added `implementation "com.mux.stats.sdk.muxstats:data-media3:1.0.0"` to gradle and added the following right after player initialization inside of `initializeSessionAndPlayer()` in `PlaybackService.kt`:
```
muxStats = player.monitorWithMuxData(
            context = baseContext,
            envKey = ENV_KEY,
            customerData = CustomerData()
    )
```

**Comment by daytime-em** [comment]
Hi, thanks for your report, and your reproduction steps! 

I was able to reproduce this issue and I should have a fix sometime next week.

**Comment by daytime-em** [comment]
@khorn4 This should be fixed in v1.0.1 of the SDK. Please reach out if you see any issues!
